tests/integration/test_fim/test_files/test_basic_usage/test_basic_usage_realtime_unsupported.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+
+import pytest
+import re
+from wazuh_testing import global_parameters
+from wazuh_testing.fim import generate_params, regular_file_cud, LOG_FILE_PATH, callback_num_inotify_watches, \
+                              detect_initial_scan, callback_ignore_realtime_flag, CHECK_ALL, REQUIRED_ATTRIBUTES
+from wazuh_testing.tools import PREFIX
+from wazuh_testing.tools.configuration import load_wazuh_configurations, check_apply_test
+from wazuh_testing.tools.monitoring import FileMonitor
+
+# Marks
+
+
+pytestmark = [pytest.mark.darwin, pytest.mark.sunos5, pytest.mark.tier(level=0)]
+
+# variables
+
+
+directory_str = os.path.join(PREFIX, 'dir')
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_conf_check_realtime.yaml')
+test_file = "testfile.txt"
+
+# configurations
+
+
+conf_params = {'TEST_DIRECTORIES': directory_str, 'MODULE_NAME': __name__}
+parameters, metadata = generate_params(extra_params=conf_params, modes=['scheduled'])
+configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
+
+
+# fixtures
+
+
+@pytest.fixture(scope='module', params=configurations)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+# tests
+
+
+@pytest.mark.parametrize('folder', directory_str)
+@pytest.mark.parametrize('file', [test_file])
+def test_realtime_unsupported(folder, file, get_configuration, configure_environment, file_monitoring,
+                              restart_syscheckd):
+    """ Check if the current OS platform falls to the scheduled mode when realtime isn't avaible.
+
+    Params:
+        folder (str): Name of the folder under PREFIX.
+        file (str): Name of the file that will be created under folder.
+        get_configuration (fixture): Gets the current configuration of the test.
+        configure_environment (fixture): Configure the environment for the execution of the test.
+        restart_syscheckd (fixture): Restarts syscheck.
+        check_realtime_mode_failure (fixture): Try to catch the initial realtime warning about ignoring the realtime  \
+            flag event and then waits for the initial FIM scan event.
+    """
+    realtime_flag_timeout = 60
```

Comment:
This variable should be placed at the beginning with the rest of the variables.

Reply:
Done in [commit](https://github.com/wazuh/wazuh-qa/pull/1474/commits/1b7ea0a207fdaf1ba67ee56d13785ac575b1ae0a)